See https://github.com/linbox-team/fflas-ffpack/issues/166
diff --git a/configure.ac b/configure.ac
index 5b46b18..c559dca 100644
--- a/configure.ac
+++ b/configure.ac
@@ -307,7 +307,6 @@ CXXFLAGS="${CXXFLAGS} ${BLAS_CFLAGS}"
 CXXFLAGS="${CXXFLAGS} ${CUDA_CFLAGS}"
 CXXFLAGS="${CXXFLAGS} ${PARFLAGS}"
 #removing possible duplicates of -std=gnu++11 and -fabi-version=6
-CXXFLAGS=`echo "${CXXFLAGS}" |xargs -n1 | sort -u|xargs`
 
 AC_SUBST(CXXFLAGS)
 
